src/admin/__tests__/fetchOffsets.spec.js

```diff
@@ -71,6 +78,66 @@ describe('Admin', () => {
       expect(offsets).toEqual([{ partition: 0, offset: '13', metadata: null }])
     })
 
+    test('returns consumer group offsets for all topics', async () => {
+      await admin.setOffsets({
+        groupId,
+        topic: topicName,
+        partitions: [{ partition: 0, offset: 13 }],
+      })
+      await admin.setOffsets({
+        groupId,
+        topic: anotherTopicName,
+        partitions: [{ partition: 0, offset: 23 }],
+      })
+      await admin.setOffsets({
+        groupId,
+        topic: yetAnotherTopicName,
+        partitions: [{ partition: 0, offset: 42 }],
+      })
+
+      const offsets = await admin.fetchOffsets({
+        groupId,
+      })
+
+      // There's no guarantee for the order of topics so we compare sets to avoid flaky tests.
+      expect(new Set(offsets)).toEqual(
+        new Set([
+          {
+            topic: yetAnotherTopicName,
+            partitions: [{ partition: 0, offset: '42', metadata: null }],
+          },
+          { topic: anotherTopicName, partitions: [{ partition: 0, offset: '23', metadata: null }] },
+          { topic: topicName, partitions: [{ partition: 0, offset: '13', metadata: null }] },
+        ])
+      )
```

Comment:
That's kind of neat. We use the package [`jest-extended`](https://github.com/jest-community/jest-extended) though, which has a matcher exactly for this, called [`toIncludeSameMembers`](https://github.com/jest-community/jest-extended#toincludesamemembersmembers).

```suggestion
      expect(offsets).toIncludeSameMembers([
        {
          topic: yetAnotherTopicName,
          partitions: [{ partition: 0, offset: '42', metadata: null }],
        },
        { topic: anotherTopicName, partitions: [{ partition: 0, offset: '23', metadata: null }] },
        { topic: topicName, partitions: [{ partition: 0, offset: '13', metadata: null }] },
      ])
```

Reply:
Similarly for the test below.